toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
No, it's one of the [0-9] strings. Try about:support, you should see `1/1 (Enabled by default` if you have one open window on Nightly (e10s enabled).